### PR TITLE
update key and val to one option in get_win_disk_vol

### PR DESCRIPTION
**Pull request by suqinhuang**
sometimes need to get device info with regular expression

e.g virtio-win-*

Signed-off-by: Suqin Huang <shuang@redhat.com>

ID: 1299366

### CODE REVIEW

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -2617,7 +2617,7 @@ def get_win_disk_vol(session, key="VolumeName", val="WIN_UTILS"):
 
     :return: volume ID.
     """
-    cmd = "wmic logicaldisk where (%s='%s') get DeviceID" % (key, val)
+    cmd = "wmic logicaldisk where (%s like '%s') get DeviceID" % (key, val)
     output = session.cmd(cmd, timeout=120)
```
Comment (xutian) [inline virttest/utils_misc.py]: def get_win_disk_vol(session, condition):
       cmd = "wmic logicaldisk where (%s) get DeviceID" % condition
       output = session.cmd(cmd, timeout=120)

File: virttest/utils_misc.py [hunk 2]
```diff
@@ -2607,7 +2607,7 @@ def selinux_enforcing():
     return utils_selinux.is_enforcing()
 
 
-def get_win_disk_vol(session, key="VolumeName", val="WIN_UTILS"):
+def get_win_disk_vol(session, key="VolumeName = 'WIN_UTILS'"):
     """
```
Comment (xutian) [inline virttest/utils_misc.py]: replace key to condition looks more meaningful

File: virttest/utils_misc.py [hunk 3]
```diff
@@ -2635,7 +2635,7 @@ def get_winutils_vol(session, label="WIN_UTILS"):
 
     :return: volume ID.
     """
-    return get_win_disk_vol(session, key="VolumeName", val="WIN_UTILS")
+    return get_win_disk_vol(session, condition="VolumeName='WIN_UTILS'")
```
Comment (xutian) [inline virttest/utils_misc.py]: condition="VolumeName='%s'" % label

**Comment by xutian** [comment]
Please update docstring too ...

**Comment by CongSmile** [comment]
@suqinhuang 

Could you please also update other cases which call get_win_disk_vol()?

Thanks.

**Comment by suqinhuang** [comment]
no case use get_win_disk_vol()

**Comment by xutian** [comment]
looks good to me.  merged. thansk @suqinhuang　and @CongSmile

**Pull request merged**